lms/djangoapps/ccx/views.py

```diff
@@ -142,18 +159,20 @@ def create_ccx(request, course):
             for vertical in sequential.get_children():
                 override_field_for_ccx(ccx, vertical, hidden, True)
 
-    url = reverse('ccx_coach_dashboard', kwargs={'course_id': course.id})
+    ccx_id = CCXLocator.from_course_locator(course.id, ccx.id)
+    url = reverse('ccx_coach_dashboard', kwargs={'course_id': ccx_id})
     return redirect(url)
 
 
 @ensure_csrf_cookie
 @cache_control(no_cache=True, no_store=True, must_revalidate=True)
 @coach_dashboard
-def save_ccx(request, course):
+def save_ccx(request, course, ccx=None):
```

Comment:
No.  We are creating override values for the course using the ccx id.  We can take apart a ccx locator and get those values, but the combination of needing to do that here and in several other dashboard views makes using the `coach_dashboard` decorator a DRYer implementation, I think.